assets/css/_map_page.scss

```
@@ -19,6 +19,10 @@ $map-page-search-panel-width: 24rem;
   padding: 0 1.5625rem;
 }
 
+.c-map-page .c-vehicle-properties-card {
+  margin: 1em;
```

nit/suggestion(non-blocking): because it's a layout property, I think this should be done with `px` instead.

Open to reasons why not to though!

---

Can you remind me why layout properties should be done with `px` instead? I'm trying to remember from past conversations and found [this thread](https://mbta.slack.com/archives/C047ZT8ACP5/p1674152194007119). The linked comment includes the following, but I may be missing some important context or later discussion/resources

> 4. Yes. rem for paddings and margins. Unlike borders and box shadows, these DO affect the readability, and should grow with the font size.

---

Thanks for digging that up, I guess I misremembered about which unit!
Based on that, are you alright with setting it to `1rem`?
```suggestion
  margin: 1rem;
```

---

I think we have used px for some sizing / layout of map controls in the past, but the reason for that was specifically that px is what Leaflet used so it was useful to use common units for things that had to align with built-in Leaflet components.